gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemProvider.java

```diff
@@ -562,7 +562,13 @@ public void createDirectory(Path dir, FileAttribute<?>... attrs) {
     final CloudStoragePath cloudPath = CloudStorageUtil.checkPath(dir);
     checkNotNull(filter);
     String prefix = cloudPath.toString();
-    final Iterator<Blob> blobIterator = storage.list(cloudPath.bucket(), Storage.BlobListOption.prefix(prefix), Storage.BlobListOption.fields()).iterateAll();
+    final Iterator<Blob> blobIterator = storage.list(
+        cloudPath.bucket(),
+        Storage.BlobListOption.prefix(prefix),
+        Storage.BlobListOption.currentDirectory()
+        // this breaks the listing (bug?)
+        //Storage.BlobListOption.fields()
```

Comment:
Happy to hear that:)

This formatting:
```java
final Iterator<Blob> blobIterator = storage.list(cloudPath.bucket(),
  Storage.BlobListOption.prefix(prefix), Storage.BlobListOption.fields()).iterateAll();
```
Rather than the one with each param on a newline.